web_programming/imdb_top.py

```diff
@@ -0,0 +1,39 @@
+import helper
+import requests
+from bs4 import BeautifulSoup
+
+
+def validate_input(movie_count):
+    try:
+        if not movie_count:
+            raise UnboundLocalError("Count must not be empty")
+        int_count = int(movie_count)
+        return int_count
+    except (UnboundLocalError, ValueError, TypeError) as e:
+        with helper.disable_exception_traceback():
+            raise (e)
+
+
+def imdb_top(imdb_top_n):
+    """
+    Args:
+        imdb_top_n: Movie count provided by user. Integer value.
+    Returns:
+        None
+    """
```

Comment:
Please look at other contributions in this repo to see how they do [type hints](https://docs.python.org/3/library/typing.html) and [doctests](https://docs.python.org/3/library/doctest.html).

Reply:
Please check now.